audition/regrets.py

```diff
@@ -44,39 +47,45 @@ def regrets_for_rule(
             selection_rule_args (dict) Arguments that the given selection rule
                 will accept as keyword arguments
 
-        Returns: (list) for each train end time, the distance between the
-            model group chosen by the selection rule and the potential
-            maximum for the next train end time
+        Returns: (list) for each train end time, a dictionary representing the
+            model group chosen by the selection rule and its performance
+
+            Should have all keys from the DistanceFromBestTable, most
+            importantly
+            'distance_from_best_case_next_time'
+            'distance_from_best_case',
+            'raw_value'
```

Comment:
I think this is going to give the current value of the metric but you'd want the next time value (the same basis for the regret calculation) since the current value is used in picking the selection rule itself -- should have been more clear about that in my comment before!